docs/compendium/candidate.md

```diff
@@ -0,0 +1,39 @@
+# Candidate process
+
+After uploading a compendium is _not_ instantly publicly available.
+It is merely a **candidate**, because metadata still must be completed for the compendium to be valid.
+
+The following process models this intermediate state of a compendium.
+
+## Creation and view
+
+Candidates can be identified by the property `candidate`.
+It is set to `true` directly after creating a new compendium by [upload](upload.md) or [public share submission](public_share.md).
```

Comment:
@nuest Will the  property be set to `true` in the front-end or in back-end?

Reply:
Regular readers won't find that issue, tried to clarify in https://github.com/o2r-project/o2r-web-api/pull/58/commits/e74f35cf66e6e5ea788e21ac536c3f53b100cbdb